mne/commands/mne_anonymize.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+"""Anonymize .fif file.
+
+You can do for example:
+
+$ mne anonymize sample_audvis_raw.fif
+"""
+
+# Authors : Dominik Krzeminski
+
+import sys
+import mne
+
+
+def mne_anonymize(fif_fname):
+    """ Call *anonymize_info* on fif file and save.
+    Parameters
+    ----------
+    fif_fname : str
+        Raw fif File
+    """
+    raw = mne.io.read_raw_fif(fif_fname, preload=True)
+    mne.io.anonymize_info(raw.info)
+    raw.save(fif_fname, overwrite=True)
```

Comment:
you should only overwrite if required.

Reply:
should we add some prefix `anon-` before anonymised file by default then?